src/devices/gps_skytraq_s1216_sup500f8.c

```diff
@@ -669,6 +668,8 @@ gps_status_t GPS_device_init(uint8_t sampleRate, struct Serial *serial)
     size_t gps_init_status = GPS_STATUS_NOT_INIT;
 
     while(attempts-- && gps_init_status == GPS_STATUS_NOT_INIT) {
+        /* Perform low level init / reset of GPS device */
+        gps_device_lld_init();
```

Comment:
In MK3 we have a 50ms delay after releasing the reset line:  

bool gps_device_lld_reset()
{
        GPIO_ResetBits(GPS_RESET_GPIO, GPS_RESET_PIN);
        delayMs(GPS_RESET_DURATION_MS);
        GPIO_SetBits(GPS_RESET_GPIO, GPS_RESET_PIN);
        delayMs(GPS_BOOT_DURATION_MS);
        return true;
}

I think this should be ok.